unit_tests/multiply_two_numbers.py

```diff
@@ -0,0 +1,14 @@
+def multiply_two_nums(num1: int, num2: int) -> int:
+    """
+    Multiply two numbers and return the result.
+    :param num1: int : Number 1 used in the multiplication
+    :param num2: int: Number 2 used in the multiplication
+    :return: int: The result of the multiplication of num1 and num2
```

Comment:
Please refer the "reStructured text" example and conform to it.
https://realpython.com/documenting-python-code/#restructured-text-example